securedrop/i18n.py

```diff
@@ -0,0 +1,76 @@
+#
+# SecureDrop whistleblower submission system
+# Copyright (C) 2017 Loic Dachary <loic@dachary.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+from flask import request, session
+from flask_babel import Babel
+from babel import core
+
+import config
+import os
+
+LOCALES = set(['en_US'])
+babel = None
+
+
+def setup_app(app):
+    global babel
```

Comment:
So looking at this, it looks like we never use the `babel` object outside this function, so to avoid us adding more global state, I think it's safe to not save the output of `Babel(app)` anywhere. If we have to save it later, we should attach it to the `app` object itself so that it's unique per app instance to help with future testing.

Reply:
Oh, nice ! It was needed in the beginning and then never questioned it again. Removed.